point/nexstar.py

```diff
@@ -398,6 +399,65 @@ def set_time(self, timestamp=None):
         ])
         self._send_command(command)
 
+    def get_gps_lock_status(self):
+        """Get the status of GPS lock
+
+        Returns:
+            bool: True if GPS is linked (locked?), false if GPS is not linked (no lock?)
+        """
+        command = b'P' + bytes([
+            1,
+            176,  # GPS Device ID?
+            55,  # Device register?
+            0,
+            0,
+            0,
+            1,  # GPS response bytes?
+        ])
+
+        response = self._send_command(command, 1)
+
+        return bool(response[0])
+
+    def get_gps_location(self):
+        """Get the GPS mount location on Earth in geographic (latitude/longitude) coordinates.
+
+        Returns:
+            tuple of floats: A pair of angles (latitude, longitude) in signed degrees format.
+        """
+        [x_var, y_var, z_var] = self._send_command(b'P' + bytes([1, 176, 1, 0, 0, 0, 3]), 3)
+        lat = ((x_var * 65536) + (y_var * 256) + z_var) / (2 ** 24) * 360
```

Comment:
This and line 432 would cause unintentional integer division in Python 2. Even though we don't support Python 2 anymore it might be a good idea to divide by `(2. ** 24)` anyway.

Reply:
Will fix.